scripts/restore_backup_osgp_dbs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+HOST=localhost
+PORT=5432
+USER=osp_admin
+BACKUP_DIR=$HOME/backups/osgp
+LOGFILE=restore.log
+
+if [ $# -eq 0 ];then
+    echo "The first argument must be a backup (file from $BACKUP_DIR/) ..."
+	ls $BACKUP_DIR/
+    exit
+fi
+
+BACKUP_DIR=$BACKUP_DIR/$1
+
+function restore() {
+        echo "- restoring $BACKUP_DIR/$1.gz ..."
+	dropdb -h $HOST -p $PORT -U $USER --if-exists $1 >> $LOGFILE
+	createdb -h $HOST -p $PORT -U $USER $1 >> $LOGFILE
+	cat $BACKUP_DIR/$1.gz | gunzip | psql -U $USER $1 >> $LOGFILE
```

Comment:
Why is the contents of the *.gz file being printed to the standard output using `cat *.gz` and then piped into `gunzip`?
Is it not possible to use: `gunzip -d *.gz` to decompress the compressed archive?

But, since the backup was created using `pg_dump`, why not use `pg_restore` to restore the backup?
That way, the `-c` option can be used to clean the database, which means the dropdb and createdb statements are no longer needed, if the database in question already exists.
Something like this perhaps: `pg_restore -h $HOST -p $PORT -U $USER -d $1 -c $BACKUP_DIR/$1.gz`